### DISCUSSION TITLE
New containers error - "No Network"

### DISCUSSION THREAD

**Discussion by Rob-T** [tteck/Proxmox#2071] [Help Needed] [answered]
Hi,

I'm suddenly receiving 'No Network' errors when any script creates a new container. Previously, the odd time I would see "No Network" flash a couple of times at most but then it would obtain an IP. During the creation process, the script seems to time-out immediately and I get the 'No Network' error right away.

Any ideas how to fix this; lengthen the time to network time-out?

Cheers!

**Comment by tteck** [comment]
The script looks to make sure it has an IP address. It checks 10 times with 3 second pauses, so 30 seconds _should_ give enough time to receive an IP from your DHCP server.

https://github.com/tteck/Proxmox/blob/f2aed7d532d04fa29edf16c32c9329bc4398e3d5/misc/install.func#L74C1-L85C5

Are you out of assignable (DHCP) IPs?

**Reply by Rob-T** [reply]
It seems to timeout almost immediately....less than 5~6 seconds for sure. I have lots of assignable IPs remaining; well over a hundred.

**Reply by Rob-T** [reply]
<img width="589" alt="Screenshot 2023-11-18 at 4 32 39 PM" src="https://github.com/tteck/Proxmox/assets/8909512/a09d081c-ad04-4f53-9b67-4a18a82f50d8">

**Reply by tteck** [reply]
Oh, you said no network when you meant no internet.
It looks as if it's trying to use IPv6 only. You need an IPv4 IP address. Try disabling IPv6 in your router.

**Reply by Rob-T** [reply]
Thanks for the help....I disable IPV6 in the script and it now tries 10 times as it used to (but I still have no internet, which is odd and a different problem).

Much appreciate the help and all of your work on the scripts!  Cheers!

**Reply by tteck** [reply, marked as answer]
Try disabling IPv6 in your router.

**Reply by Rob-T** [reply]
Thanks...this worked. Not sure why it is suddenly causing the issue (I have dozens of containers running and this was never an issue). Regardless, thank you for the help!

**Reply by reebevoli** [reply]
> Try disabling IPv6 in your router.

Hi,

I've just realized I need IPV6 for the Matter network in my smart home, so IPV6 needs to be turned back on. 
I'm wondering if anyone has ideas why this would suddenly be an issue; 26 containers previous the the last one had zero issue connecting to the internet upon creation via their respective script. 

Thanks again!
